pkg/agent/openflow/network_policy.go

```diff
@@ -2239,7 +2239,7 @@ func (f *featureNetworkPolicy) l7NPTrafficControlFlows() []binding.Flow {
 			Done(),
 		// This generates the flow to forward the returned packets (with FromTCReturnRegMark) to stageOutput directly
 		// after loading output port number to reg1 in L2ForwardingCalcTable.
-		TrafficControlTable.ofTable.BuildFlow(priorityHigh).
+		TrafficControlTable.ofTable.BuildFlow(priorityHigh+1).
```

Comment:
What's this change for? The flow has the same match and condition as the one in `trafficControlCommonFlows`, changing priority shouldn't change anything.

Reply:
This is not a necessary change. As you mentioned, this is the same flow with the one in `trafficControlCommonFlows`.  When both L7NP and TrafficControl are enabled, the same flows will be added and it is a little weird, so I changed the priority. I think it is also okay to revert the change since we can handle duplicated flows when syncing flows to OVS.